monai/apps/auto3dseg/auto_runner.py

```diff
@@ -285,17 +277,13 @@ def __init__(
         self.ensemble = ensemble  # last step, no need to check
 
         self.set_training_params()
+        self.set_device_info()
         self.set_prediction_params()
         self.set_analyze_params()
 
-        self.save_image = self.set_image_save_transform(kwargs)
-
-        self.ensemble_method: AlgoEnsemble
-        self.ensemble_method_name: str | None = None
-
+        self.ensemble_method_name = 'AlgoEnsembleBestByFold'
         self.set_num_fold(num_fold=num_fold)
-        self.set_ensemble_method("AlgoEnsembleBestByFold")
-
+        self.kwargs = kwargs
```

Comment:
At the first sight, `kwargs` seems too general as a property in the `__init__` function. Usually we may need some logics to process the possible kwargs.

Reply:
Yes I agree kwargs is too general. kwargs was originally in __init__, I did not add it. I only added self.kwargs = kwargs because set_image_transform function needs kwargs as input, which is moved to EnsembleRunner.

Reply:
Can we keep the `set_image_save_transform` method in AutoRunner and pass the save transform to the `__init__` of `EnsembleRunner`?

The disadvantage of passing `self.kwargs` to another module `EnsembleRunner` : kwargs is a `dict` and the methods in  `EnsembleRunner` can change the values in `kwargs`, which will change `self.kwargs` and make it unsafe when more changes are built into Auto3DSeg.

Here is an example of how bug can be created:

```
class Bar:
    def __init__(self, **kwargs):
        self.kwargs = kwargs
    
    def buggy_method(self):
        self.kwargs["input"]["a"] = 1

class Foo:
    def __init__(self, **kwargs):
        self.kwargs = kwargs
    
    def update(self):
        self.bar = Bar(**self.kwargs)
        self.bar.buggy_method()

c = {"a": 0}
A = Foo(input=c)
print("before: ", c)
A.update()
print("bug occurred: ", c)
```

The output:
```
before:  {'a': 0}
bug occurred:  {'a': 1}
```